api/src/main/java/io/onetable/client/CatalogConfig.java

```diff
@@ -7,15 +7,14 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Hi @wuchunfu. 
In this PR, many changes correspond to replacing **http**://www.apache.org/licenses/LICENSE-2.0 with **https**://www.apache.org/licenses/LICENSE-2.0.
However, I looked at a few Apache projects (Airflow, Spark, etc). None of the projects have `https`. Even on the ASF 2.0 license site, the link is `http` and not `https` ([link](https://www.apache.org/licenses/LICENSE-2.0#apply)).  I also checked some source files in Apache Skywalking, and there is no https link there ([link](https://github.com/apache/skywalking/blob/master/apm-webapp/src/main/java/org/apache/skywalking/oap/server/webapp/ApplicationStartUp.java)). 

Are you sure this change is needed. Could you please share a reference?

Reply:
@ashvina Thank you for your reminder. Although I couldn't find any available examples https://www.apache.org/licenses/LICENSE-2.0 The real link protocol now is HTTPS, even for requests http://www.apache.org/licenses/LICENSE-2.0 It will also be redirected to https://www.apache.org/licenses/LICENSE-2.0 I'll just follow the usual practice, it's fixed now, PTAL, thanks